.github/workflows/clean-tsv.yml

```diff
@@ -27,14 +27,14 @@ jobs:
     - name: clean location hierarchies
       run: |
         pipenv sync
-        pipenv run ./bin/clean-tsv source-data/location_hierarchy.tsv --n-cols 4
+        pipenv run ./bin/clean-tsv source-data/location_hierarchy.tsv source-data/location_hierarchy.tsv --n-cols 4
     - name: clean annotations
       run: |
         pipenv sync
         grep -v "paper\|title" source-data/gisaid_annotations.tsv > nonpaper_annotations.tsv
-        pipenv run ./bin/clean-tsv nonpaper_annotations.tsv --n-cols 4 --no-header
+        pipenv run ./bin/clean-tsv nonpaper_annotations.tsv nonpaper_annotations.tsv --n-cols 4 --no-header
         grep "paper\|title" source-data/gisaid_annotations.tsv > paper_annotations.tsv
-        pipenv run ./bin/clean-tsv paper_annotations.tsv --n-cols 4 --sort-col 1 --no-header
+        pipenv run ./bin/clean-tsv paper_annotations.tsv paper_annotations.tsv --n-cols 4 --sort-col 1 --no-header
```

Comment:
Oh, I wrote this in error. This is the output as well as the input. Got it.

Reply:
This was just me passing the same name for the input and output files (overwriting), since I was assuming we didn't want preserve intermediate output in the github workflow.

Reply:
Yes, thank you, I eventually realized that.